build.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+SETLOCAL
+
+
+.paket\paket.bootstrapper.exe
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
+.paket\paket.exe restore
+if errorlevel 1 (
+  exit /b %errorlevel%
+)
+
+:Build
+cls
+
+SET FAKE_PATH=packages\build\FAKE\tools\Fake.exe
+SET Platform=
+
+IF [%1]==[] (
+    "%FAKE_PATH%" "build.fsx" "Default" 
+) ELSE (
+    "%FAKE_PATH%" "build.fsx" %* 
+) 
+
+if defined TEAMCITY_PROJECT_NAME goto Quit
+
+rem Loop the build script.
```

Comment:
Why should one loop the build script? It is meant to be run alongside developing?